third_party/txt/src/minikin/LineBreaker.cpp

```diff
@@ -361,7 +361,8 @@ void LineBreaker::pushBreak(int offset, float width, uint8_t hyphenEdit) {
 // with nullptr as the paint property, which will lead it to assume the width
 // has already been calculated. Used for properly breaking inline widgets.
 void LineBreaker::setCustomCharWidth(size_t offset, float width) {
-  mCharWidths[offset] = (width);
+  mCharWidths.reserve(offset + 1);
```

Comment:
If `mCharWidths` needs to be expanded, then call `resize` here to update its size in addition to its capacity.